website/react-magma-docs/src/pages/data-visualization-intro/introduction.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Getting Started
+order: 1
+---
+
+import { LeadParagraph } from '../../components/LeadParagraph';
+import { Flex, FlexBehavior, FlexAlignItems, magma } from 'react-magma-dom';
+
+<PageContent componentName="getting_started" type="data_visualization_intro">
+
+<LeadParagraph>
+  Magma charts enable you to convey precise and compelling narratives around
+  data through visually appealing and accessible visualizations. 
+</LeadParagraph>
+
+Magma Charts is based on [Carbon Charts from IBM](https://charts.carbondesignsystem.com/introduction).
+
+Get started below with implementation instructions for developers. You can also learn more about the [chart types](#) we provide, as well as [view examples](#).
+
```

Comment:
We should include somewhere here the content of this alert:

![image](https://github.com/user-attachments/assets/34b5edfe-0d5b-474d-9a92-d9876883c09f)